### DISCUSSION TITLE
What sorts of features should be demonstrated/featured?

### DISCUSSION THREAD

**Discussion by Volte6** [GoMudEngine/GoMud#86] [Q&A]
I have a number of screenshots and describe a handful of features here:

https://github.com/Volte6/GoMud/blob/master/webclient/images/README.md

However, I'm not sure what's the most compelling or interesting. I've left out quite a lot, obviously specific skills and such, but went for stuff that might catch someones eye as either an admin or a player. Stuff that's visually interesting/colorful I think helps too.

What is worth featuring though?

**Comment by windu-ant** [comment]
I guess its time for me to dive back into this and get you some suggestions. It looks great. Is there an online building interface? "edit room description" "build room north" etc. That would be another thing to feature I think if there is.

**Reply by Volte6** [reply]
There is but it's a little bit ad-hoc and risky. 

I'm gonna improve it and make it a series of prompts for more advanced stuff... right now commands like :

`build room north south` - builds an exit to the north and connects it back to this room via a "south" exit (can be any exit name). Or ommit the return name for a one way... it jjust copies the room you're in... and there are various commands to spice up the room like `room set description <a bunch of text>` etc... but so much has changed i'm sure some of the commands need some fixing up, and probably need too add new ones.

Mostly it served as a tool to build out some initial basic content quickly... and slowly over time I start offloading special functionality to script files too.

It needs some serious consideration on how to do it right, I guess.

**Reply by Volte6** [reply]
<img width="1460" alt="image" src="https://github.com/user-attachments/assets/4bfb458b-6e34-48a4-aacd-b6bf4838ff86">
